frontend/src/components/design-system/containers/CardComponent.vue

```diff
@@ -0,0 +1,91 @@
+<script setup lang="ts">
+import { computed } from 'vue';
+
+export interface Props {
+  radius?: 'small' | 'large';
+  corner?: 'top-left' | 'top-right' | 'bottom-left' | 'bottom-right';
```

Comment:
napr. tady bych ocekaval, kdyz je ten prop optional, ze kdyz tam nedam nic, tak to nebude mit corner zadnej, ale ve skutecnosti je tam nejaka default hodnota, ktera je pro toho uzivatele skryta.

Reply:
Už není optional.